sources/packages/backend/apps/api/src/services/institution-location/institution-location.service.ts

```diff
@@ -55,11 +71,7 @@ export class InstitutionLocationService extends RecordDataModelService<Instituti
       id: locationId ?? undefined,
     } as InstitutionLocation;
 
-    if (locationId) {
-      saveLocation.modifier = auditUser;
-    } else {
-      saveLocation.creator = auditUser;
-    }
+    saveLocation.creator = auditUser;
```

Comment:
Please move it inside the object block that starts on line 58.

Reply:
ok! Done. Thanks!